flux/src/actions/index.tsx

```diff
@@ -8,20 +8,14 @@ function SuspendAction(props) {
   const { resource } = props;
   const [open, setOpen] = React.useState<boolean>(false);
 
+  if(resource.jsonData.spec.suspend) {
```

Comment:
Do you need to check whether you have the `suspend` prop there? Because before you were checking.
Maybe `resource?.jsonData?.spec?.suspend`

Reply:
fixed